OrbitGl/App.h

```diff
@@ -200,6 +201,7 @@ class OrbitApp final : public DataViewFactory, public CaptureListener {
   void CrashOrbitService(orbit_grpc_protos::CrashOrbitServiceRequest_CrashType crash_type);
 
   DataView* GetOrCreateDataView(DataViewType type) override;
+  DataView* GetOrCreateSelectionCallstackDataView();
```

Comment:
[[nodiscard]]

Reply:
Maybe add it for the method above as well since you're at it

Reply:
Yes, thanks! Fixed it